hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -1455,22 +1455,22 @@ public String getName() {
     @Override
     public void clear() {
         records.clear();
-        onClear();
     }
 
-    protected void onClear() {
+    public void clearAndSendInvalidation() {
```

Comment:
Maybe this method ought to be part of the `ICacheRecordStore` interface? Please also check this in light of `clear` method usage on EE side.